### DISCUSSION TITLE
Supabase version 2.0.5 don't have `auth.user()` method

### DISCUSSION THREAD

**Discussion by rimzzlabs** [supabase/supabase#10093] [Questions]
Hi, I don't get this property on newer supabase version, any idea how to get user?

supabase-js version: `@^v2.0.5`

**Comment by GaryAustin1** [comment]
This is a document of some of the more common changes.
https://supabase.com/docs/reference/javascript/upgrade-guide#get-the-logged-in-user
Edit: the above seems to have a formatting issue.
https://supabase.com/docs/reference/javascript/upgrade-guide

```
const {

  data: { session },

} = await supabase.auth.getSession()

const { user } = session
```

**Reply by rimzzlabs** [reply]
thanks, I forgot to read migration guide to version 2
